web/models/invitation.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Entice.Web.Invitation do
+  use Ecto.Model
+
+  schema "invitations" do
+    field :email, :string
+    field :key, :string
+    field :createdat, Ecto.DateTime, default: Ecto.DateTime.local
```

Comment:
You may be able to define a post-insert-hook to set this value automatically when new fields are created, check the ecto docs...